### PR TITLE
Splitted guide on AWS and Azure

### PR DESCRIPTION
**Pull request by fiunchinho**
Towards https://github.com/giantswarm/giantswarm/issues/12706

We will support creating clusters directly from the CP API on Azure.

### CODE REVIEW

**Review by MartaD — COMMENTED** [review]
AWESOME! 🚀

File: src/content/guides/creating-clusters-via-crs-on-azure/index.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+All the tenant clusters, created with release version {{% first_azure_nodepools_version %}} and newer, are managed as [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) in the Control Plane.
+
+At a high-level, the Control Plane API is used to manage the following CRs:
+
+- [Cluster](/reference/cp-k8s-api/clusters.cluster.x-k8s.io/) - represents a Kubernetes cluster excluding worker nodes.
+- [MachinePool](/reference/cp-k8s-api/machinepools.exp.cluster.x-k8s.io/) - represents a node pool.
+
+The CRs above then reference provider specific implementations. In our case, for clusters on Azure, they are:
+
+- [AzureCluster](/reference/cp-k8s-api/azureclusters.infrastructure.cluster.x-k8s.io/) - represents a tenant cluster.
```
Comment (tuommaki) [inline src/content/guides/creating-clusters-via-crs-on-azure/index.md:24]: Should there also be a mention of `AzureMachine` for tenant cluster masters?

File: src/content/guides/creating-clusters-via-crs-on-azure/index.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+All the CRs, mentioned above, have strict spec and important requirements to be considered valid.
+There is very limited CR validation available in the Control Plane for now.
+Therefore, if you create a CR with wrong field values, that can result in a broken tenant cluster.
+That's why we've developed a simple [CLI utility](https://github.com/giantswarm/kubectl-gs), which helps to template valid CRs.
+
+The utility supports rendering CRs:
+
+- Tenant clusters:
+    - `Cluster` (API version `cluster.x-k8s.io/v1alpha3`)
+    - `AzureCluster` (API version `infrastructure.cluster.x-k8s.io/v1alpha3`)
```
Comment (tuommaki) [inline src/content/guides/creating-clusters-via-crs-on-azure/index.md:170]: `AzureMachine` for tenant master?

Comment (fiunchinho) [reply src/content/guides/creating-clusters-via-crs-on-azure/index.md:177]: Added

File: src/content/guides/creating-clusters-via-crs-on-azure/index.md (hunk 1 above)
Comment (fiunchinho) [reply src/content/guides/creating-clusters-via-crs-on-azure/index.md:24]: It's not referenced by `Cluster` or `MachinePool`. But I'll add it down below.

**Review by tuommaki — APPROVED** [review]
LGTM

**Pull request merged**